qa/L0_perf_ts/test.sh

```diff
@@ -38,6 +38,9 @@ if [ ! -z "$TEST_REPO_ARCH" ]; then
     REPO_VERSION=${REPO_VERSION}_${TEST_REPO_ARCH}
 fi
 
+sudo apt-key del 7fa2af80
```

Comment:
We should not need to use sudo here. Also add a comment why this is needed, and file a follow up ticket to remove this when possible.

Reply:
Thanks,I moved away `sudo` call on given statement. 
For the issue you mentioned, should it be public?

Reply:
Just mention `DLIS-<ticket-id>` in the comment. It will be fine.

Reply:
Thank you, - proposed changes were added